scipy/signal/tests/test_signaltools.py

```diff
@@ -109,6 +109,13 @@ def test_input_swapping(self):
         assert_array_equal(convolve(big, small, 'valid'),
                            out_array[1:3, 1:3, 1:3])
 
+    def test_convolve_method(self):
+        x = np.random.rand(10)
```

Comment:
make sure this is deterministic by setting the random seed if it isn't already

    np.random.seed(1234)

https://github.com/numpy/numpy/blob/master/doc/TESTS.rst.txt#tests-on-random-data